src/JobRunner.php

```diff
@@ -80,6 +80,11 @@ public function run()
                 // How many jobs are runned
                 array_push($this->jobs, $task);
 
+                if (!$task->saveRunningFlag(true) && $task->getRunType() == 'single') {
+                    $this->cliWrite(($task->name ?: 'Task') . ' is single run task and one instance already running.', 'red');
```

Comment:
$this->cliWrite('Failed: ' . $task->name, 'red');

Reply:
changes done

Reply:
You can't set name of task, the system assign one automatically

Reply:
https://github.com/daycry/cronjob/pull/20/commits/f9603cdd671709a949c146d1de112e7142dde146

already done in above commit please check